src/config.js

```diff
@@ -1,7 +1,4 @@
 export default {
```

Comment:
Since the config has moved to env vars should we rename `src/config.js` to e.g. `src/abi/oracle.abi.js`?